akka-docs/release-train-issue-template.md

```diff
@@ -32,7 +32,7 @@ Key links:
 
 - [ ] Check [API](https://doc.akka.io/api/akka-core/$VERSION$/) documentation
 - [ ] Check [reference](https://doc.akka.io/libraries/akka-core/$VERSION$/) documentation. Check that the reference docs were deployed and show a version warning (see section below on how to fix the version warning).
-- [ ] Check the release on https://repo.akka.io/TOKEN/secure/com/typesafe/akka/akka-actor_2.13/$VERSION$/akka-actor_2.13-$VERSION$.pom
+- [ ] Check the release `coursier resolve com.typesafe.akka:akka-actor_2.13:$VERSION$ -r $(grep -oE 'https://[^"]+' ~/.sbt/1.0/credentials.sbt | head -1)` 
```

Comment:
I suggest that we remove this step and use PR validation of akka-dependencies as verification instead. Perhaps move that step up before updating doc links.

everybody might not have the tokens setup in same way
I don't even have `coursier` installed

Reply:
What about if it doesn't require another third party tool, this seems to work as well:

```
mvn dependency:get -Dartifact=com.typesafe.akka:akka-actor_2.13:2.10.11
```